DistValetDriver.php

```diff
@@ -39,7 +43,10 @@ public function isStaticFile($sitePath, $siteName, $uri)
      * @param  string  $uri
      * @return string
      */
-    public function frontControllerPath($sitePath, $siteName, $uri)
+    /**
```

Comment:
There are two docblocks for this method now. There should be one

Reply:
Oops. Fixed.